validator/accounts/v2/wallet.go

```diff
@@ -269,6 +325,29 @@ func (w *Wallet) ReadPasswordForAccount(accountName string) (string, error) {
 	return string(password), nil
 }
 
+// ReadFileForAccount from the wallet's accounts directory.
+func (w *Wallet) ReadFileForAccount(accountName string, fileName string) ([]byte, error) {
```

Comment:
All of these can likely be abstracted to reduce code duplication, but want to save that for a next PR